challenges/Ayush/medium-task-3-create-3d-card-hover-effect/index.html

```
@@ -0,0 +1,246 @@
+    transition: transform 0.1s;
+}
+
+/* Calculate rotation:
+   rotateX = (mouse-y - card-center-y) / card-height * rotation-amount
+   rotateY = (mouse-x - card-center-x) / card-width * -rotation-amount
+*/</code></pre>
+            </div>
+
+            <div class="code-block">
+                <h3>JavaScript Mouse Tracking</h3>
+                <pre><code>card.addEventListener('mousemove', (e) => {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)